### ISSUE TITLE
Column header ellipsis doesn't show with themes

### ISSUE THREAD

**Issue by ataft** [ag-grid/ag-grid#1850]
As shown below, ellipsis do not show up for any of the themes (except 'none'):
![image](https://user-images.githubusercontent.com/11670864/29790507-fc90a2bc-8bee-11e7-8732-fab3d11bccfb.png)

It looks like this is due to the "float" css, as shown below:

```
.ag-bootstrap .ag-numeric-header .ag-header-cell-label>span {
    float: right;
}

.ag-bootstrap .ag-header-cell-label>span {
    float: left;
}
```

**Comment by petyosi** [comment]
Thank you, logged as AG-712.

**Comment by petyosi** [comment]
The issue should be fixed in 13.0.1, about to be released today.
